src/Text/Pandoc/Writers/Markdown.hs

```diff
@@ -327,8 +351,15 @@ blockToMarkdown' opts (Div attrs ils) = do
   contents <- blockListToMarkdown opts ils
   variant <- asks envVariant
   return $
-    case () of
-         _ | isEnabled Ext_fenced_divs opts &&
+     case () of
+         _ | variant == Markua -> 
+                   case () of
+                        () | classes' == ["blurb"] -> prefixed "B> " contents <> blankline
+                           | classes' == ["aside"] -> prefixed "A> " contents <> blankline
+                           -- | necessary to enable option to create a bibliography
+                           | (take 3 (T.unpack id')) == "ref" -> contents <> blankline
+                           | otherwise -> mempty
```

Comment:
I don't think we should be completely omitting the contents of Divs that don't have these special classes!
Instead, just fall back to emitting the contents.

Reply:
we change it so write the contents